pkgs/development/python-modules/stups-tokens/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv
+, buildPythonPackage
+, fetchpatch
+, fetchPypi
+, requests
+}:
+
+buildPythonPackage rec {
+  pname = "stups-tokens";
+  version = "1.1.19";
+  doCheck = false;
+  propagatedBuildInputs = [
+    requests
+  ];
+  src = fetchPypi {
```

Comment:
generally I like to see src come before any build inputs... as it aligns more closely to the order in which the expression gets evaluated. Also build inputs can be quite long in some cases